webpack-babel.config.js

```diff
@@ -0,0 +1,20 @@
+var path = require('path');
+
+module.exports = {
+  output: {
+    libraryTarget: 'umd',
```

Comment:
Actually answering the `why UMD` question: because it works and because we use UMD in `webpack.config.js`. But let's follow the instructions you linked.